### PR TITLE
Comment NL33

### PR DESCRIPTION
**Pull request by Aurige**
Additional TransportOrganisations and GroupOfOperators to LINE

### CODE REVIEW

**Comment by skinkie** [comment]
This is not in the documentation nor in master. But I would rather propose a breaking change. Allowing _multiple_ OperatorRef's and AuthorityRef's, instead of the current _xs:choice_. Hence lets have a discussion prior to merging. cc @hennoit

**Review by ue71603 — APPROVED** [review]

**Comment by skinkie** [comment]
@ue71603 I still want to have a discussion on this one.

File: xsd/netex_part_1/part1_networkDescription/netex_line_version.xsd [hunk 1]
```diff
@@ -395,11 +395,19 @@ Rail transport, Roads and Road transport
...
+					</xsd:annotation>
+				</xsd:element>
+				<xsd:element name="additionalTransportOrganisations" type="transportOrganisationRefs_RelStructure" minOccurs="0">
+					<xsd:annotation>
+						<xsd:documentation>Additional TRANSPORT ORGANISATIONs for LINE (authority, operator). +v1.1</xsd:documentation>
+					</xsd:annotation>
+				</xsd:element>
+				<xsd:element ref="GroupOfOperatorsRef" minOccurs="0"/>
+			</xsd:choice>
 			<xsd:element name="otherModes" type="modeRefs_RelStructure" minOccurs="0">
```
Comment (ue71603) [inline xsd/netex_part_1/part1_networkDescription/netex_line_version.xsd:411]: @skinkie  so you want to have additionalOperators and additionalAuthorities and no choice?
This is fine for me as well.

Comment (skinkie) [reply xsd/netex_part_1/part1_networkDescription/netex_line_version.xsd:411]: Actually, I want to know why additionalOperators is preferred over "multiple operators".

**Review by klement-MENTZ — APPROVED** [review]

**Comment by ue71603** [comment]
Christophe: Already done in other PR.

**Comment by trurlurl** [comment]
Removing "needs documentation update" label since closed without having been merged.

**Pull request closed without merging**